app/views/layouts/admin.html.haml

```diff
@@ -7,18 +7,17 @@
     %title= content_for?(:title) ? yield(:title) : t('titles.admin', brand: t('titles.brand'))
     %meta{ content: content_for?(:description) ? yield(:description) : "Admin Interface for #{t('titles.brand')}", name: 'description' }
     = favicon_link_tag 'CHARMcareCFavicon.png', rel: 'icon', type: 'image/png'
-    = stylesheet_link_tag 'https://fonts.googleapis.com/css?family=Lato:400,700'
+    = stylesheet_link_tag 'http://fonts.googleapis.com/css?family=Lato:400,700', rel: 'stylesheet', type: 'text/css'
     = stylesheet_link_tag 'admin/application', media: 'all'
-    = stylesheet_pack_tag 'application'
+    = stylesheet_link_tag "tailwind", "inter-font", "data-turbo-track": "reload"
```

Comment:
where does this file come from? I'm expecting to see something in app/assets/stylesheets/tailwind.scss for this to work, does the tailwind cli create this file on the fly?

Reply:
hmmm i'm not sure, I think some tailwind tutorial told me to put it there. I'll try removing it and see if that breaks it

Reply:
It's necessary. I think this is the main stylesheet link to the minified css that tailwind build on the fly. I'm not 100% on how tailwind works under the hood tbh, it's a beast